src/XrdOuc/XrdOucUtils.cc

```diff
@@ -1400,5 +1400,14 @@ int XrdOucUtils::getModificationTime(const char *path, time_t &modificationTime)
     return statRet;
 }
 
+void XrdOucUtils::trim(std::string &str) {
+    // Trim leading non-letters
+    while( str.size() && !isgraph(str[0]) ) str.erase(str.begin());
+
+    // Trim trailing non-letters
+
+    while( str.size() && !isgraph(str[str.size()-1]) )
```

Comment:
This is a silly optimization but when you think about it, you want to trim the end first as std:string will not move any characters but merely adjust the size and place a zero at the end.  Then when you trim the front there are potentially less characters to move around. I say silly because you likely will never notice this optimization. I couldn't resist though.